lib/bungie_client.dart

```diff
@@ -274,16 +274,16 @@ class BungieClient {
             .map((skull) => skull['displayName']));
   }
 
-  /// Returns the progress on Moments of Triumph as a completion percentage for
+  /// Returns the progress on Age of Triumph as a completion percentage for
```

Comment:
I have a website which displays that information in a leaderboard, so it's still useful.